src/data/analyses.js

```diff
@@ -0,0 +1,121 @@
+var MAP = {
```

Comment:
we should extract this to a common place, not sure how. I'm worried about adding a new analysis and forgetting to update this.